pkg/curatedpackages/bundle.go

```diff
@@ -103,3 +105,17 @@ func (b *BundleReader) getActiveController(ctx context.Context) (*packagesv1.Pac
 	}
 	return obj, nil
 }
+
+func (b *BundleReader) UpgradeBundle(ctx context.Context, controller *packagesv1.PackageBundleController, newBundle string) error {
+	controller.Spec.ActiveBundle = newBundle
+	controllerYaml, err := yaml.Marshal(controller)
+	if err != nil {
+		return err
+	}
+	params := []string{"create", "-f", "-", "--kubeconfig", b.kubeConfig}
+	_, err = b.kubectl.CreateFromYaml(ctx, controllerYaml, params...)
```

Comment:
can we have something like `CreateFromYaml(ctx context.Context, yaml []byte, kubeconfig string) (bytes.Buffer, error)`?

Reply:
Any reason for this? The current approach provides flexibility to pass in any arguments.

Reply:
Because the method is called `CreateFromYaml`. It doesn't make a lot of sense that we need to pass "create" as part of the params. The method already knows it needs to run a create command.